sample-apps/aws-xray-sdk-go-sample/main.go

```diff
@@ -0,0 +1,69 @@
+// Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the License. A copy of the License is located at
```

Comment:
I think you should add below lines here in file headers.

```
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: MIT-0
```

Reply:
Sure